include/configcat/configcatlogger.h

```diff
@@ -15,13 +15,15 @@ class ConfigCatLogger {
         hooks(hooks) {
     }
 
-    void log(LogLevel level, const std::string& message) {
+    void log(LogLevel level, int eventId, const std::string& message) {
         if (hooks && level == LOG_LEVEL_ERROR) {
             hooks->invokeOnError(message);
         }
 
         if (logger) {
-            logger->log(level, message);
+            std::stringstream ss;
+            ss << "[" << eventId << "] " << message;
+            logger->log(level, ss.str());
```

Comment:
It's a little bit faster if we don't create a stringstream here:
```suggestion
            logger->log(level, "[" + std::to_string(eventId) + "] " + message);
```